### DISCUSSION TITLE
real-esrganv2模型转换结果严重偏色

### DISCUSSION THREAD

**Discussion by tumuyan** [xinntao/Real-ESRGAN#254] [General]
原图

![原图.jpg](https://user-images.githubusercontent.com/3126801/153791974-8d18c6d0-b32e-4519-952d-2ee4e9c8ecad.jpg)

原模型

![Real-ESRGAN-anime_0214_104259.png](https://user-images.githubusercontent.com/3126801/153791796-379b7597-8f11-4c25-b439-1a4818945379.png)

v2模型

![Real-ESRGANv2-anime_0214_104311.png](https://user-images.githubusercontent.com/3126801/153791998-b46bb7a2-cc05-40a1-a506-554270ac94a5.png)

可以看到文字的颜色明显发生了变化

**Comment by AkoZ** [comment]
yep here about [esrgan updated](https://colab.research.google.com/drive/1O8kr-iyRll5eQsaOH-vQIc3ImEKWsvF-?usp=sharing#scrollTo=D6HDAdt3tE15) .. also it's all ok for it well tested on a VQgan image.. 
* to say it operates well in OPERA browser ! no need of fullTiringMemory->Chrome ;)
it seems fast too ! so to say wp ! well worked and a big thank YOU @xinntao

**Reply by xinntao** [reply]
Thanks

**Comment by xinntao** [comment]
> 原图
> 
> ![原图.jpg](https://user-images.githubusercontent.com/3126801/153791974-8d18c6d0-b32e-4519-952d-2ee4e9c8ecad.jpg)
> 
> 原模型
> 
> ![Real-ESRGAN-anime_0214_104259.png](https://user-images.githubusercontent.com/3126801/153791796-379b7597-8f11-4c25-b439-1a4818945379.png)
> 
> v2模型
> 
> ![Real-ESRGANv2-anime_0214_104311.png](https://user-images.githubusercontent.com/3126801/153791998-b46bb7a2-cc05-40a1-a506-554270ac94a5.png)
> 
> 可以看到文字的颜色明显发生了变化

@tumuyan 

谢谢报告这个问题，我们更新了v3模型：参见  https://github.com/xinntao/Real-ESRGAN/blob/master/docs/anime_video_model.md
和  https://github.com/xinntao/Real-ESRGAN/blob/master/docs/anime_comparisons.md
v3 模型解决了偏色问题，并且比 v2有更好的纹理表现。

当然和anime 6B的纹理细节不能比，还有差距，毕竟v3是一个很小的模型

**Reply by tumuyan** [reply]
感谢！v3模型效果已经非常棒了，并且速度与v2接近
